website/docs/cloud-docs/vcs/private.mdx

```diff
@@ -0,0 +1,85 @@
+
+- HCP Terraform agent v1.15.0 or newer.
+- Network connectivity between the HCP Terraform agent and private VCS provider.
+- Outbound network connectivity from the HCP Terraform agent to HCP Terraform. Refer to [networking requirements](/terraform/cloud-docs/agents/requirements#networking-requirements) for more information.
+
+## Configure the agent pool and agent
+
+Before you start your HCP Terraform agent, you must create an agent pool and token. Refer to [Create an Agent Pool](/terraform/cloud-docs/agents/agent-pools#create-an-agent-pool) for a list of steps to create an agent pool.
+
+To configure your HCP Terraform agent with private VCS support, you must set the following environment variables in the environment you run the agent. You can also set these environment variables with optional command-line flags.
+
+| Variable | Value | Command-Line Flag |
+| -------- | ----- | ----------------- |
+| `_TFC_AGENT_REQUEST_FORWARDING_ENABLED` | `true` | |
+| `TFC_AGENT_REQUEST_FORWARDING` | `true` | `-request-forwarding` |
+| `TFC_AGENT_ACCEPT` | A comma-separated list of values that  must include `ingress`. Refer to [Install and Run Agents](/terraform/cloud-docs/agents/agents#accept) for more information. | `-accept` |
+
+Below is an example command to start the HCP Terraform agent:
+
+<CodeBlockConfig hideClipboard>
+
+```bash
+$ export TFC_AGENT_TOKEN=<...>
+
+$ _TFC_AGENT_REQUEST_FORWARDING_ENABLED=true \
+  tfc-agent \
+  -name private-vcs-test-01 \
+  -request-forwarding=true \
+  -accept "ingress" \
+  -log-level "debug"
+```
+
+</CodeBlockConfig>
+
+## Configure the private VCS connection
+
+To create a private VCS connection, complete the following steps:
+
+1. In HCP Terraform, navigate your organization's **Settings** page.
+1. Under **Version Control**, click **Providers**.
+1. Click **Add a VCS provider**.
+1. Select your VCS Provider.This must be a self-managed provider, such as GitHub Enterprise.
+1. Under **VCS Access**, select “Private”.
+1. Select the agent pool that you created for private VCS.
+1. The remainder of the setup follows the standard VCS setup workflow. Refer to the [standard VCS setup instructions](/terraform/cloud-docs/vcs) for provider specific instructions.
+
+
+## Troubleshooting
+
+There are two sources of information to help diagnose issues with Private VCS and the HCP Terraform agent:
+
+- HCP Terraform agent logs
+  - These display events as seen by the HCP Terraform agent.
+  - They can aid in diagnosing errors between the HCP Terraform agent and the VCS provider during VCS ingestion, as well as any issues occurring during request forwarding.
+- VCS events in HCP Terraform (Organization **Settings > Version Control > Events**)
+  - VCS events include errors that have resulted from making requests for private VCS during setup of the VCS Connection, and during normal operation.
+  - This will not include messages during normal operations of private VCS.
```

Comment:
since we mentioned 2, this looks better as a numbered list

```suggestion
1. HCP Terraform agent logs
  - These display events as seen by the HCP Terraform agent.
  - They can aid in diagnosing errors between the HCP Terraform agent and the VCS provider during VCS ingestion, as well as any issues occurring during request forwarding.
1. VCS events in HCP Terraform (Organization **Settings > Version Control > Events**)
  - VCS events include errors that have resulted from making requests for private VCS during setup of the VCS Connection, and during normal operation.
  - This will not include messages during normal operations of private VCS.
```